Src/IronPython/Compiler/Parser.cs

```diff
@@ -2401,40 +2308,41 @@ struct WithItem {
             return l;
         }
 
-        private Expression ParseTestListAsExpr() {
+        // testlist: test (',' test)* [',']
+        private Expression ParseTestList() {
             if (!NeverTestToken(PeekToken())) {
-                var expr = ParseExpression();
+                var expr = ParseTest();
                 if (!MaybeEat(TokenKind.Comma)) {
                     return expr;
                 }
 
-                return ParseTestListAsExpr(expr);
+                return ParseTestList(expr);
             } else {
-                return ParseTestListAsExprError();
+                return ParseTestListError();
             }
-        }
 
-        private Expression ParseTestListAsExpr(Expression expr) {
-            List<Expression> l = new List<Expression>();
-            l.Add(expr);
+            Expression ParseTestList(Expression expr) {
```

Comment:
It's hard to tell with the formatting, but is this a nested method now? If not, why was the `private` removed.

Reply:
No need to rename, it was just hard to tell from the diff.